src/scancode/cli.py

```diff
@@ -131,6 +131,29 @@ def print_about(ctx, param, value):
 
     scancode -f json -l -c samples/zlib/ > scan.json
 
+Scan a directory while ignoring a single file. Print scan results to stdout as JSON:
```

Comment:
eg. if I use `--ignore "foo*bar"` and I have a path `jogn/foo/fubar/bar` will this be ignored? This is what you need to clarify for the help examples.